packetbeat/protos/http/http_parser.go

```diff
@@ -169,22 +193,24 @@ func (*parser) parseHTTPLine(s *stream, m *message) (cont, ok, complete bool) {
 	return true, true, true
 }
 
-func parseResponseStatus(s []byte) (uint16, string, error) {
-	debugf("parseResponseStatus: %s", s)
+func parseResponseStatus(s []byte) (uint16, []byte, error) {
+	if isDebug {
+		debugf("parseResponseStatus: %s", s)
+	}
 
-	p := bytes.Index(s, []byte(" "))
+	p := bytes.IndexByte(s, ' ')
 	if p == -1 {
-		return 0, "", errors.New("Not beeing able to identify status code")
+		return 0, nil, errors.New("Not beeing able to identify status code")
```

Comment:
Not new, but "beeing" and spelled wrong and can just be dropped form the message. It can be fixed at other time and PR.